packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.1.0]
+
+### Added
+
+- feat: add updateNft fct to update Nft metadata whenever its requested. ([#4008](https://github.com/MetaMask/core/pull/4008))
```

Comment:
A few things:

- We don't use conventional commits in this repo, so `feat:` isn't necessary (the "Added" header says that it's a feature already)
- It looks like this PR not only added a new method to NftController, but also added a new type which is being exported, so we should note that as well.
- The name of the method added is `updateNftMetadata`, not `updateNft`.

What about:

``` suggestion
- Add `updateNftMetadata` method to `NftController` to update metadata for the requested NFTs ([#4008](https://github.com/MetaMask/core/pull/4008))
- Add `NftUpdate` type to represent the NFT metadata results within `updateNftMetadata` ([#4008](https://github.com/MetaMask/core/pull/4008))
```

Alternatively, did we mean to export `NftUpdate`? I don't see it being used in the NftController tests. We could take that out if so.

Reply:
Thanks @mcmire ! Your suggestion looks great 🙏   
The exported type is not used you are right. Should i close this PR, create a new one that removes the export of the type then create another release branch?

Reply:
You can create a new one that removes the export, but don't need to close this one. Once your PR is merged you can just sync the new changes from `main` and then they will be included in the release.

Reply:
Okay great so now this can become:

``` suggestion
- Add `updateNftMetadata` method to `NftController` to update metadata for the requested NFTs ([#4008](https://github.com/MetaMask/core/pull/4008))
```

Reply:
I ve pushed this update :D